.gitignore

```diff
@@ -232,3 +232,6 @@ tag-mapping.json
 
 # Temporary files generated by import sorting linter.
 *.isorted
+
+# Temporary files generated by conda.
+.conda/
```

Comment:
```suggestion
```

Reply:
Reminder to revert this change @chrislevn

Reply:
Or if you think it's a necessary change in general for Ray developers, you can open a separate PR for it